.github/workflows/release-serverless-build.yaml

```diff
@@ -0,0 +1,98 @@
+name: release serverless build
+on:
+  push:
+    tags:
+      - '^v?\d+\.\d+\.\d+(?:-.*)?$'
+permissions:
+  id-token: write
+  contents: read
```

Comment:
I would suggest to keep comments about permissions ([source](https://github.com/kyma-project/test-infra/blob/main/cmd/image-builder/README.md#workflow-permissions))

Reply:
I will add it in next PR